Makefile.kokkos

```diff
@@ -1,5 +1,9 @@
 # Default settings common options.
 
+ifneq ($(KOKKOS_USE_DEPRECATED_MAKEFILES), 1)
+  $(error Makefile support is deprecated. Set KOKKOS_USE_DEPRECATED_MAKEFILES=1 to silence this error.)
```

Comment:
We can also advise to switch to CMake.

Reply:
@cedricchevalier19 Would you suggest a new wording for the =error message?